ui/src/components/search/ModelNameSearch.js

```diff
@@ -18,17 +19,17 @@ const modelNameSearch = (
 ) => {
   return (
     <SidebarSection title="Search by Model Name">
-      {/* <QuickSearch
-        {...rest}
-        searchLowercase={true}
-        placeholder="Enter model name ..."
-        searchTextDelimiters={[',']}
-        InputComponent={TextInput}
-        LoadingIcon={
-          <Spinner fadeIn="none" name="circle" color="#a9adc0" style={{ width: 15, height: 15 }} />
-        }
-        // DropdownItemComponent={QuickSearchDropdownItem}
-      /> */}
+      <QuickSearch
+      // {...rest}
+      // searchLowercase={true}
+      // placeholder="Enter model name ..."
+      // searchTextDelimiters={[',']}
+      // InputComponent={TextInput}
+      // LoadingIcon={
+      //   <Spinner fadeIn="none" name="circle" color="#a9adc0" style={{ width: 15, height: 15 }} />
+      // }
+      // DropdownItemComponent={QuickSearchDropdownItem}
+      />
```

Comment:
Props to review -- what's now internal to Arranger and can be removed vs what needs to be adapted to v3

Reply:
note for posterity: we talked about this topic elsewhere, and discussed how these props are now handled through the "theming engine", and needn't be passed in directly to the component like this.